Report/Fluvoxamine_evaluation_report.md

```diff
@@ -0,0 +1,403 @@
+| based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/releases/tag/v1.0 |
+| OSP Version                                     | 10.0                                                         |
+| Qualification Framework Version                 | 2.3                                                          |
+
+This evaluation report and the corresponding PK-Sim project file are filed at:
+
+https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/
+# Table of Contents
+  * [1 Introduction](#1-introduction)
+  * [2 Methods](#2-methods)
+    * [2.1 Modeling Strategy](#21-modeling-strategy)
+    * [2.2 Data](#22-data)
+    * [2.3 Model Parameters and Assumptions](#23-model-parameters-and-assumptions)
+  * [3 Results and Discussion](#3-results-and-discussion)
+    * [3.1 Final input parameters](#31-final-input-parameters)
+    * [3.2 Diagnostics Plots](#32-diagnostics-plots)
+    * [3.3 Concentration-Time Profiles](#33-concentration-time-profiles)
+      * [3.3.1 Model Building](#331-model-building)
+      * [3.3.2 Model Verification](#332-model-verification)
+  * [4 Conclusion](#4-conclusion)
+  * [5 References](#5-references)
+  * [6 Glossary](#6-glossary)
+# 1 Introduction
+The presented PBPK model of fluvoxamine has been developed to be used in a PBPK Drug-Drug-Interactions (DDI) network with fluvoxamine as an inhibitor of CYP2C19 and CYP1A2.
+
+Fluvoxamine is a selective serotonin reuptake inhibitor. After oral administration, fluvoxamine-maleate is absorbed moderately fast with peak plasma concentrations reached at approx. 2-4 hours post-dose. Approximately 77% of the drug is bound to plasma proteins. The concentration-time profile elicits a bi-phasic shape ([Perucca 1994](#5-References)). Fluvoxamine is extensively metabolised via CYP2D6 and CYP1A2. Plasma concentrations increase non-proportionally with increasing doses, suggesting (partially) saturable metabolism. Fluvoxamine is a strong inhibitor of CYP2C19 and CYP1A2 ([FDA DDI Labeling](#5-References)), and a weak inhibitor of CYP3A (AUCR <2) ([Lam 2003](#5-References)). Only a very minor part of the dose is recovered unchanged in the urine.
+
+# 2 Methods
+
+
+## 2.1 Modeling Strategy
+The general workflow for building an adult PBPK model has been described by Kuepfer et al. ([Kuepfer 2016](#5-References)). Relevant information on the anthropometry (height, weight) was gathered from the respective clinical study, if reported. Information on physiological parameters (e.g. blood flows, organ volumes, hematocrit) in adults was gathered from the literature and has been incorporated in PK-Sim® as described previously ([Willmann 2007](#5-References)). The  applied activity and variability of plasma proteins and active processes that are integrated into PK-Sim® are described in the publicly available 'PK-Sim® Ontogeny Database Version 7.3' ([PK-Sim Ontogeny Database Version 7.3](#5-References)).
+
```

Comment:
`([PK-Sim Ontogeny Database Version 7.3](#5-References)).`
Please always use lower case in the links. We use automaed link checkers in PBPK-Model_lib and Qualification-Report-Lib. Unfortunately one of them can only properly handle links in lower case. Thus for the link abover we will get a false alarm. If you change it to `([PK-Sim Ontogeny Database Version 7.3](#5-references)).`  -then all good.